docs/DafnyRef/Types.md

```diff
@@ -2758,13 +2712,67 @@ a ``...`` which means to copy the signature from
 (if `M0.F` does not provide one). It can also add `ensures`
 clauses.
 
+If a function definition does not have a body, the program that contains it may still be verified.
+The function itself has nothing to verify.
+However, any calls of a body-less function are treated as unverified assumptions by the caller,
+asserting the preconditions and assuming the postconditions.
+Because body-less functions  are unverified assumptions, programs containing them may not be compiled.
```

Comment:
```suggestion
Because body-less functions are unverified assumptions, Dafny cannot compile them and will complain if called by [`dafny translate`, `dafny build` or even `dafny run`](https://dafny.org/latest/DafnyRef/DafnyRef#256-using-dafny-from-the-command-line)
```